docs/pages/guides/using-eslint.mdx

```diff
@@ -167,6 +167,29 @@ To install Prettier in your project:
 
 ### Setup
 
+<Tabs>
+
+<Tab label="Flat Config">
+
+To integrate Prettier with ESlint, update your **eslint.config.js**:
```

Comment:
It seems the new flat config, by default, will create the **eslint.config.js** file instead of **.eslintrc.js**.

Let's update the ESLint > Setup section to update the file name:

![CleanShot 2025-04-06 at 22 13 43](https://github.com/user-attachments/assets/a2812249-f305-4b35-bb63-174686109376)